src/main/java/school/lemon/changerequest/java/Employees/Employees.java

```diff
@@ -0,0 +1,40 @@
+package school.lemon.changerequest.java.Employees;
+
+/**
+ * Created by User on 05.12.2016.
+ */
+public class Employees {
+    public Employees(String nameE,int workHoursE){
```

Comment:
Not sure it's a good way to pass worked hours via constructor. I'd prefer add method like `work(int hours)` or simple `setWorkedHours(int hours)`.